liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/extensions/thin/SpringBootMavenThinPluginMojo.java

```diff
@@ -67,12 +77,17 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         try {
             File sourceFatJar = getTargetFile();
             thin(sourceFatJar);
+        } catch (SpringBootThinException e) {
+            throw new MojoExecutionException(
+                    "Execution of plugin failed because it requires an executable archive. Configure the repackage goal of the spring-boot-maven-plugin before, to create an executable archive",
```

Comment:
Plugin execution failed because the application archive is not an executable archive. The repackage goal of the spring-boot-maven-plugin must be configured to run first in order to create the required executable archive.